html/php-pre-challenge2/index.php

```diff
@@ -2,9 +2,23 @@
 $array = explode(',', $_GET['array']);
 
 // 修正はここから
-for ($i = 0; $i < count($array); $i++) {
+// バブルソート
+// 参考：https://mikaduki.info/webprogram/php/1120/
 
+//並べ替え自体を行うセット回数
+for ($i = 0; $i < count($array); $i++)
```

Comment:
count関数がループの度に毎回実行されてしまうので、計算結果を変数に入れることでcount関数の実行を1回にしましょう

あと外側のループの回数を1回減らせます

```suggestion
$length = count($array);
for ($i = 0; $i < $length - 1; $i++) {
```